packages/components/CONTRIBUTING.md

```diff
@@ -276,7 +276,7 @@ export function useCard( props ) {
 }
 
 //=========================================================================
-// Simplified snippet from `packages/components/src/card/card/component.js`
+// Simplified snippet from `packages/components/src/card/card/component.ts`
```

Comment:
I found one more instance that needs renaming:

https://github.com/WordPress/gutenberg/blob/8f82fbc970a4250fee4dd7ac333aa890f69af16d/packages/components/CONTRIBUTING.md?plain=1#L316